src/components/Home.jsx

```diff
@@ -0,0 +1,12 @@
+import React, { Component } from "react";
+
+class Home extends Component {
+  render() {
+    return (
+      <div className="container">
+        <h3>Author's Haven</h3>
```

Comment:
@joshNic Please squash commits, and also maybe use tabs instead of spaces, another option would be to edit the `eslintrc.json` to allow for spaces but I would not advise it if we are to stick to a particular style guide.